lib/pender/exception/item_not_available.rb

```diff
@@ -0,0 +1,5 @@
+module Pender
+  module Exception
+    class ItemNotAvailable < StandardError; end
+  end
+end
```

Comment:
Question: What do you think about grouping all exceptions under the same module, instead of having one file for each?